PurchasesCoreSwift/Attribution/AdClientProxy.swift

```diff
@@ -38,18 +37,15 @@ class FakeAdClient: NSObject {
 
 }
 
-// TODO (post-migration): Make all the things internal again.
-@objc(RCAdClientProxy)
-public class AdClientProxy: NSObject {
+class AdClientProxy {
```

Comment:
Same as https://github.com/RevenueCat/purchases-ios/pull/754/files#r692916082

Reply:
This one will need to be manually tested because we **_might_** need to inherit from NSObject for the proxy stuff to work.